packages/zsa/README.md

```diff
@@ -23,18 +23,39 @@ npm i zsa zsa-react zsa-react-query zsa-openapi zod
 
 ## Documentation
 
-View the full documentation and examples on [zsa.vercel.app](https://zsa.vercel.app/docs/introduction).
+View the full documentation and examples on [zsa.vercel.app](https://zsa.vercel.app/docs/introduction). The documentation includes:
+
+- Getting started guide
+- In-depth explanations of core concepts
+- Examples for building fully typesafe server actions
+
+If you're looking to learn how to get the most out of `zsa`, [the documentation](https://zsa.vercel.app/docs/introduction) is the best place to start.
```

Comment:
It seems redundant to include the same link twice, and the information could be more concise in a single statement as it were before. When checking the ZSA documentation, the purpose is already clear, so listing reasons for visiting feels unnecessary. Additionally, adding these details may increase maintenance for ZSH, as it could lead to outdated or incorrect information about what the ZSA documentation covers.